src/08_comprehensions.py

```diff
@@ -11,13 +11,16 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
+for i in range(1,6):
+    y.append(i)
 print (y)
```

Comment:
All of these solutions result in the correct output which is great!

However, this exercise calls for the use of list comprehensions which will become really important and useful tool in CS. They provide a more concise way of creating lists.

---

Recall how list-comprehensions are constructed and try to re-work you're each of your solutions: 

```
#  output           i for each number in range 1 through 6
#    ^              ^
y = [i          for i in range(1, 6)]
```

You can think of list comprehensions in this way: that each `i` from the for-loop becomes a "output" or new element in the resulting list.

```
y = [i for i in range(1, 6)]

print(y)
# [1, 2, 3, 4, 5]
```

---

**Note**: That the for-loop can iterate through ranges and as well as iterable objects like lists, 

and that you can manipulate your output _before_ it becomes the new element in the resulting list:

```
nums = [1, 2, 3, 4, 5]

#   output            n for each element in nums
#     ^               ^
y = [n*2          for n in nums]

print(y)
# [1, 4, 6, 8, 10]
```